pkg/runbook/config.go

```diff
@@ -1,14 +1,16 @@
-package runbookconfig
+package runbook
 
 import (
 	"github.com/giantswarm/micrologger"
 	"k8s.io/client-go/kubernetes"
 )
 
-type RunbookInput map[string]string
+type Input map[string]string
 
-type RunbookConfig struct {
+type Config struct {
```

Comment:
moved and renamed this so it's `runbook.Config`, more aligned with go conventions, instead of `runbookconfig.RunbookConfig` where the names are repeated.